### DISCUSSION TITLE
1 click Prompt Generator

### DISCUSSION THREAD

**Discussion by cdosrunwild** [v8hid/infinite-zoom-automatic1111-webui#90] [Ideas]
1 click Prompt Generator with a random option, and how about we all share our prompts by adding a share my prompt with everyone and this will update the list with a click, to update live list...

**Comment by GeorgLegato** [comment]
at least you can "Export your prompt" clicking the button below table to download a json file. You or others can "Import Promps" from that json.
So, you can ask someGPT to extend such json. You can take a look in the izHelpers-directors where you can have a look into the jsonschema file for ultimate details

**Comment by GeorgLegato** [comment]
here is an example json
{
  "prompts": {
    "data": [
      [
        0,
        "new york"
      ],
      [
        2,
        "berlin"
      ],
      [
        3,
        "shanghai"
      ],
      [
        5,
        "a Verdant canopy"
      ]
    ]
  },
  "negPrompt": "frames, border, edges, borderline, text, character, duplicate, error, out of frame, watermark, low quality, ugly, deformed, blur, bad-artist",
  "prePrompt": "roof top shot",
  "postPrompt": "style of my favorite artist"
}
